src/display_buffer.hh

```diff
@@ -143,7 +144,7 @@ private:
     AtomList  m_atoms;
 };
 
-DisplayLine parse_display_line(StringView line);
+DisplayLine parse_display_line(StringView line, HashMap<String, DisplayLine> m_atoms = HashMap<String, DisplayLine>{});
```

Comment:
m_atoms shoud just be atoms, and I think atom is not such a good name, I'd suggest built_ins, or named_values. Also, the default value can just be written `{}`, no need to repeat the type.